packages/navigator/src/browser/navigator-model.ts

```diff
@@ -71,7 +71,9 @@ export class FileNavigatorModel extends FileTreeModel {
 
     protected async createRoot(): Promise<TreeNode | undefined> {
         if (this.workspaceService.opened) {
-            const workspaceNode = WorkspaceNode.createRoot();
+            const workspaceNode = this.workspaceService.tryGetRoots().length > 1
```

Comment:
I mean that there can be multi-root workspace with only one root, then the top node should have a name of a workspace file and first child should be a root. Have a look how it is done in VS Code.

Reply:
yes I see now :) thankfully the PR is only a draft at the moment until I iron out the details